org.cubocore.CoreImage.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --share=network
```

Comment:
What does it need network access for?

Reply:
My bad accidentally added. It does not require any network.

Reply:
The app didn't require any network though. But having little warning with `network`
`Qt: Session management error: Could not open network socket`

Will it be a problem if `network` used?

Reply:
No, I believe it's about something else.

Reply:
The warning you mean?

Reply:
Yeah, I don't remember the details anymore. It's not critical anyway, should be a warning.

Reply:
OK Fine.